modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -1,21 +1,27 @@
-= Create an OData v4 API
+= Building, Implementing, and Testing an OData V4 API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: creating-an-odata-api-with-apikit.adoc
 
-The OData MySQL example shows you how to use an HTTP-based data service to query a MySQL database.
+You can query a MySQL database with the xref:creating-an-odatav4-api-with-apikit.adoc#apikit-for-odata-4-implementation-example[OData MySQL example] by using an HTTP-based data service.
 
-== Prerequisites
+== Before You Begin
 
-The following software is required to create and use an OData REST API with APIkit:
+You must install the following software to create and use an OData REST API with APIkit:
 
 * OData Plugin
 * Mule runtime engine 4.3.0 or later
 * Anypoint Studio 7.9.0 or later
 
-== Scaffold an OData v4 API Using APIkit for OData
+Additionally, you must download the link:{attachmentsdir}/apikit-odata4-example.jar[OData 4 implementation example].
+
+== Scaffold an OData V4 API Using APIkit for OData
 
 . In Studio, select *File* > *New* > *Mule Project*.
 . In the *Project Name* field, type the name of your Mule project.
 . Click *Finish*.
-. In `+src/main/resources/api+`, create a new file with a _.csdl.xml_ extension. +
+. In `+src/main/resources/api+`, create a new file with a `.csdl.xml` extension. +
```

Comment:
Global - put the + on a line by itself. This affects the spacing. See https://confluence.internal.salesforce.com/display/MTDT/Formatting+with+Asciidoc#FormattingwithAsciidoc-ListContinuation(akaIndentedParagraphs)

Reply:
OK! Thanks!